ci-scripts/build-tx-query.sh

```diff
@@ -0,0 +1,15 @@
+set -e
+
+APP_NAME=tx-query2-enclave-app
+RUNNER_NAME=tx-query2-app-runner
+PRIVATE_KEY_PATH=chain-tx-enclave/tx-query/enclave/Enclave_private.pem
+
+# Build enclave app
+RUSTFLAGS="-Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3,+pclmul -Clinker=gold -Clink-args=-shared" cargo build --package $APP_NAME --target x86_64-fortanix-unknown-sgx
```

Comment:
is this script ok? a few days ago, it seemed this config was problematic, so one should still go with rust-lld and just add `CFLAGS="-gz=none"`?

Reply:
or is this script used?

Reply:
This is not used anymore. I'll delete this.